### ISSUE TITLE
New PR on feat-deno branch

### ISSUE THREAD

**Issue by erfanGharib** [telegraf/telegraf#1920]
While contributing to the Telegraf repository, [MKRhere](https://github.com/MKRhere) suggested [here](https://github.com/telegraf/telegraf/pull/1915#issuecomment-1866934885) that I create my PR on the feat-deno branch. However, the issue arises as the feat-deno branch contains existing bugs and is 101 commits behind the v4 branch. What should I do now? Should fix the bugs in feat-deno, or is there another recommended approach?

![image](https://github.com/telegraf/telegraf/assets/79592676/699af971-12c1-4fec-8566-d27545e6078a)

**Comment by MKRhere** [comment]
Right, feat-deno branch uses Deno, so you should install the Deno extension for VSCode.
